hasura/metadata/databases/default/tables/public_reactions.yaml

```diff
@@ -26,14 +26,21 @@ insert_permissions:
       check:
         activity:
           organization:
-            circles:
-              users:
-                _and:
-                  - deleted_at:
-                      _is_null: true
-                  - profile:
-                      id:
+            _or:
+              - members:
+                  _and:
+                    - profile_id:
                         _eq: X-Hasura-User-Id
+                    - deleted_at:
+                        _is_null: true
+              - circles:
```

Comment:
once we launch org members, are we going to want to keep this OR or make these only org_member dependent? @levity 

I think just org_member dependent seems simpler... but not until it launches can we make that changes ,right?